database/cassandraSeed.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+###################################################
+# Bash script to create database and seed
+###################################################
+date
+# Variable Definitions
+# Path to directory bash script is living
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Database Variable Definitions
+# PASSWORD=""
```

Comment:
Looks like you don't have a password, but I'm not sure, anyway pushing this even as a comment to github, can be not a good practice, probably use a global var in separate file and add it to .gitignore